glib/example/vala/sqlite.vala

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+int main (string[] args) {
+    try {
+    	var database = new GADBC.Database ();
+    	database.set_option ("driver", "adbc_driver_sqlite");
+  	database.set_option ("uri", "test.db");
+    	database.init ();
+    	var conn = new GADBC.Connection ();
+    	conn.init (database);
```

Comment:
```suggestion
    	var database = new GADBC.Database();
    	database.set_option("driver", "adbc_driver_sqlite");
  	database.set_option("uri", "test.sqlite");
    	database.init();
    	try {
    	        var connection = new GADBC.Connection();
    	        connection.init(database);
    	        connection.release();
    	} catch (GLib.Error error) {
    	    	GLib.message("Error: %s", e.message);
    	}
    	database.release();
```

Reply:
Ok. Because database creating and initialization requires to handle its errors, database and connection initialization will have its own error handling.